snippets/csharp/System.Net.Http/SocketsHttpHandler/ConnectCallback/program.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+
+class HttpClientHandler_SecureExample
+{
+    // <Snippet1>
+    static async Task Main()
+    {
+        // Create an HttpClientHandler object and set to use default credentials
```

Comment:
```suggestion
```

I assume this has been copypasted from another example, I don't think the construction line needs an explanation in this one.

Reply:
yes this is copy from another example.

Reply:
updated